integration/utils/contLoadGenerator.go

```diff
@@ -36,23 +41,27 @@ const tokensPerTransfer = 0.01     // flow testnets only have 10e6 total supply,
 // ContLoadGenerator creates a continuous load of transactions to the network
 // by creating many accounts and transfer flow tokens between them
 type ContLoadGenerator struct {
-	log                  zerolog.Logger
-	loaderMetrics        *metrics.LoaderCollector
-	tps                  int
-	numberOfAccounts     int
-	flowClient           access.Client
-	serviceAccount       *flowAccount
-	flowTokenAddress     *flowsdk.Address
-	fungibleTokenAddress *flowsdk.Address
-	favContractAddress   *flowsdk.Address
-	accounts             []*flowAccount
-	availableAccounts    chan *flowAccount // queue with accounts available for workers
-	workerStatsTracker   *WorkerStatsTracker
-	workers              []*Worker
-	stopped              bool
-	loadType             LoadType
-	follower             TxFollower
-	availableAccountsLo  int
+	log                        zerolog.Logger
+	loaderMetrics              *metrics.LoaderCollector
+	tps                        int
+	numberOfAccounts           int
+	flowClient                 access.Client
+	serviceAccount             *flowAccount
+	flowTokenAddress           *flowsdk.Address
+	fungibleTokenAddress       *flowsdk.Address
+	favContractAddress         *flowsdk.Address
+	accounts                   []*flowAccount
+	availableAccounts          chan *flowAccount // queue with accounts available for workers
+	workerStatsTracker         *WorkerStatsTracker
+	workers                    []*Worker
+	stopped                    bool
+	loadType                   LoadType
+	follower                   TxFollower
+	availableAccountsLo        int
+	maxConstExecTxSizeInByte   uint
```

Comment:
the number of arguments passed into NewContLoadGenerator is getting a bit out of hand.  could you group these fields into a sub-struct? e.g.,

```
type ConstExecTx struct {
    maxSizeInByte
    authAccNum
    argSizeInByte
    payerKeyCount
}
```

Reply:
done